ports/alienpogs/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "alienpogs.zip",
+  "items": [
+    "Alien Pogs From Outer Space.sh",
+    "alienpogs"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Alien Pogs From Outer Space",
+    "porter": [
+      "TheLaggyPyro"
+    ],
+    "desc": "This Bullet Heaven Survival game was made for GBJam11 from Sept 15-25 2023. .",
+    "desc_md": "v",
+    "inst": "Download any version of the game from  [Itch.io](https://meredosia.itch.io/alien-pogs-from-outer-space), extract the zip file and copy the .rpg file into the the  alienpogs port folder.",
```

Comment:
Portmaster app does not support markdown language.
Only the inst_md field for the website.

So inst short and snappy for the PortMaster App and inst_md for the Game Details page with full markdown support.